src/codeview.jl

```diff
@@ -167,14 +167,15 @@ function cthulhu_typed(io::IO, debuginfo::Symbol,
         Returns(nothing)
     end
     if !isnothing(remarks)
+        sort!(remarks)
         function postprinter(io::IO, @nospecialize(typ), used::Bool)
             _postprinter(io, typ, used)
             haskey(io, :idx) || return
             idx = io[:idx]::Int
-            for (pc, remark) in remarks
-                if pc == idx
-                    printstyled(io, ' ', remark; color=:light_black)
-                end
+            firstind = searchsortedfirst(remarks, idx=>"")
+            for i in firstind:lastindex(remarks)
```

Comment:
can we further limit this iteration space to `firstind:searchsortedfirst(remarks, idx+1=>"")`?

Reply:
ah I see.